sound/soc/sof/topology.c

```diff
@@ -16,6 +16,7 @@
 #include "ops.h"
 
 #define COMP_ID_UNASSIGNED		0xffffffff
+
```

Comment:
different patch

Reply:
@plbossat did you mean to separate out the topology parts into a separate patch?

Reply:
no, move the newline somewhere else

Reply:
@plbossart ahh ok. I've removed it. I dont know how it sneaked in.